manifests/pipecd/templates/deployment.yaml

```diff
@@ -227,6 +227,9 @@ spec:
         {{- include "pipecd.selectorLabels" . | nindent 8 }}
         app.kubernetes.io/component: ops
     spec:
+      {{- if .Values.serviceAccount.create -}}
+      serviceAccountName: {{ include "pipecd.fullname" . }}-server
+      {{- end }}
```

Comment:
Follow this [comment](https://github.com/pipe-cd/pipecd/pull/3535/files#r851000700)

https://github.com/pipe-cd/pipecd/blob/master/manifests/piped/templates/deployment.yaml#L22-L24
https://github.com/pipe-cd/pipecd/blob/master/manifests/piped/templates/_helpers.tpl#L76-L85